composer.json

```diff
@@ -13,14 +13,12 @@
     "require" : {
         "php" : ">=5.3",
         "symfony/framework-bundle": ">=2.3,<3.0",
-        "ftrrtf/rollbar-notifier": "~1.0"
+        "ftrrtf/rollbar-notifier": "~1.0",
+        "twig/extensions": "^1.1"
```

Comment:
@ftrrtf definitely, but there are tests for it. So I will leave it as suggested package and maybe add it as dev dependency?